app/models/developer.rb

```diff
@@ -51,6 +51,10 @@ class Developer < ApplicationRecord
     joins(:location).where(locations: {utc_offset:})
   end
 
+  scope :filter_by_country, ->(country) do
```

Comment:
Interestingly, we are using both singular and plural for these filters.

* `filter_by_role_types`
* `filter_by_role_levels`
* `filter_by_utc_offset` - singular
* `filter_by_country` - singular

I'd like to consolidate these to a single tense. Since it is less work, lets go with plural.

Can you please change this to `filter_by_countries` and above to `filter_by_utc_offsets`?